src/manual/containers-parallel.adoc

```diff
@@ -58,6 +59,65 @@ public void paralletTest() {
 }
 ----
 
-So the normal test action processing would be to execute one action after another. As the first action is a sleep of five seconds, the whole test processing would stop and wait for 5 seconds. Things are different inside the parallel container. Here the descending test actions will not wait but execute at the same time.
+The normal test action processing would be to execute one action after another. As the first action is a sleep of
+five seconds, the whole test processing would stop and wait for 5 seconds. Things are different inside the parallel
+container. Here the descending test actions will not wait but execute at the same time.
 
-NOTE: Note that containers can easily wrap other containers. The example shows a simple combination of sequential and parallel containers that will archive a complex execution logic. Actions inside the sequential container will execute one after another. But actions in parallel will be executed at the same time.
\ No newline at end of file
+If you are using this container to send or receive messages, you have to use the unique correlation id of the
```

Comment:
Both async and parallel containers mention message correlation. Maybe this could be restructured in the future?

Reply:
Yeah, absolutely!
A review of the documentation is on the roadmap for 2019. That's why I did not put so much effort into this addition, because it is planned to change in anyways.